superset/views/dashboard/views.py

```diff
@@ -44,8 +44,7 @@ class DashboardModelView(
     datamodel = SQLAInterface(models.Dashboard)
 
     @action("mulexport", __("Export"), __("Export dashboards?"), "fa-database")
-    @staticmethod
-    def mulexport(items):
+    def mulexport(self, items):  # pylint: disable=no-self-use
```

Comment:
What's the reasoning behind this change?

Reply:
It's related with this already merged PR: https://github.com/apache/incubator-superset/pull/8942

the `@action` decorator creates a methods attribute on the method, this is defeated by making the method static